### DISCUSSION TITLE
spec.toss() run chain of request

### DISCUSSION THREAD

**Discussion by smitsarang8** [pactumjs/pactum#367] [Q&A]
When i have multiple scenarios in Feature file

then i observe spec.toss() executes request in following manner

scenario 1 is executed once
scenario 2 is executed twice
scenario 3 is executed thrice

is there any solution to not run chained request? in cascade manner and only run one request?

**Comment by ASaiAnudeep** [comment]
Have you tried reseting the spec after each scenario.

Example: https://github.com/pactumjs/pactum-cucumber-boilerplate/blob/master/features/support/steps.js#L7
